1_The_Physical/Level_1_Stateless_Guess_Point/1_Arrange_Act_Assert_Backwards/2_2_Stats_Calculator/src/index.spec.ts

```diff
@@ -1,5 +1,36 @@
+import { calculateStats } from './index';
 
 describe('stats calculator', () => {
+    describe('When an array of numbers is provided', () => {
```

Comment:
It would be more readable for future debugging to validate each of the method's features separately with a few examples (rather than validating all features with one example)

For example, for the feature of finding the min value:
describe -> finds the min value
<each test validates a different example>